nss-copper-dragonflies/BangazonSite
# Mr order

# Description

Given the user is authenticated
And the user is viewing their open order
When the user clicks the Complete Order button
Then the user should be presented with a view that allows them to select a payment type for the order
Given the user selects a payment option
When the user clicks the Done button
Then the payment type must be added to the order
And the user will be presented with a confirmation/thank you screen

User can also add a product by clicking the [add to order] button. The user will be directed to their order with all the current info.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

# Testing Instructions

fetch --all
git checkout mr-order
click view cart to see if your user has any orders, if you do complete the order by adding a payment type.
if you do not have an open order go to product categories and click on the add to order button. This should redirect you to your new open order



# Checklist:
